docs/stable/2-account.md

```diff
@@ -219,3 +221,13 @@ All messages sent are encrypted with the public key of the destination and signe
 ## Security Considerations
 
 -
+
+## Changelog
+
+### Version 0.3
+
+Released `TODO`
```

Comment:
Don't forget this one

Reply:
I've got it on my Todo list. Once this is merged I'll reference the commit hash in an update PR.